src/modules/EnvironmentVariables/EnvironmentVariables/EnvironmentVariablesXAML/Views/MainPage.xaml

```diff
@@ -2,14 +2,14 @@
     x:Class="EnvironmentVariables.Views.MainPage"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:controls="using:CommunityToolkit.WinUI.Controls"
     xmlns:converters="using:EnvironmentVariables.Converters"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:i="using:Microsoft.Xaml.Interactivity"
     xmlns:ic="using:Microsoft.Xaml.Interactions.Core"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:models="using:EnvironmentVariables.Models"
-    xmlns:toolkitConverters="using:CommunityToolkit.WinUI.Converters"
+    xmlns:tk8controls="using:CommunityToolkit.WinUI.Controls"
+    xmlns:tkconverters="using:CommunityToolkit.WinUI.Converters"
```

Comment:
@niels9001 This is what I meant: I _can_ keep `xmlns:converters="using:EnvironmentVariables.Converters"`, but **only because "PowerToys...Converters" is not present**. If, at any point in the future, in any file, both versions are needed, then we have to look at this again.

Reply:
But those "PowerToys converters" live in Settings? How would we use those in a module? There's no relation or UI code sharing between modules in Settings or across modules, separate .projs

In case we'd move all custom converters PowerToys.sln to a dedicated library so they can be re-used, then updating the namespace would make sense.

Reply:
Ohhhh... I think I get it now! They'll never mix! 🤓💡
Okay that solves that question. Will update things later 👍🏻